sources/packages/web/src/views/student/CheckValidSINBanner.vue

```diff
@@ -2,7 +2,7 @@
   <banner
```

Comment:
i know its not part of the PR, this is not a  view it a component, it should be `component` folder